CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## Pending changes
 
+- [#197](https://github.com/bumble-tech/appyx/pull/197) – **Breaking change**: `ParentNodeView` does not implement plugin anymore. Node instance is retrieved via LocalComposition.   
```

Comment:
Maybe mention that `AbstractParentNodeView` and `AppyxParentViewTestRule.kt` have been removed?